Sources/WalletSdk/StorageManager.swift

```diff
@@ -89,14 +78,13 @@ class StorageManager: NSObject {
       return nil
    }
 
-   // Method: remove()
-   //    Remove a key/value pair. Removing a nonexistent key/value pair is not an error.
-   //
-   // Arguments:
-   //    key - the name of the file
-   //
-   // Returns:
-   //    A boolean indicating success; at present, there is no failure path, but this may change in the future.
+   /// Remove a key/value pair. Removing a nonexistent key/value pair is not an error.
+   ///
+   /// - Parameters:
+   ///    - key: the name of the file
+   ///
+   /// - Returns:
+   ///    - A boolean indicating success; at present, there is no failure path, but this may change in the future.
```

Comment:
Does it work with the blank line in between the docs and the function?

Reply:
It does, but I’ve pushed a change to remove the space.